bundles/core/org.eclipse.smarthome.core.ephemeris/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0 http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="system:ephemeris">
+		<parameter name="weekendDays" type="text" required="true" multiple="true">
+			<default>Saturday,Sunday</default>
+			<label>Week-end Days</label>
+			<description>List of usual non workable days, separated by comma.</description>
```

Comment:
I'd remove the ", separated by comma" as this might depend on how a UI decides to let the user select from the different options.